packages/mantine/src/theme/plasmaCSSVariablesResolver.ts

```diff
@@ -23,6 +23,14 @@ export const plasmaCSSVariablesResolver: CSSVariablesResolver = (theme) => {
             '--mantine-color-gray-light-hover': theme.colors.gray[1],
             '--mantine-color-warning-filled': theme.colors.yellow[4],
             '--mantine-color-placeholder': theme.colors.gray[4],
+            '--mantine-color-default-hover': theme.colors.gray[1],
+            '--mantine-radius-default': theme.radius.sm,
```

Comment:
The border radius doesn't look quite right

Current (4px)
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/c0131e05-5e98-4fed-81a9-d96be7a9b253" />

Wanted (8px)
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/48bc561c-b93b-4c07-b7f4-019c790da7c6" />

Reply:
Thank you, I had the [discussion](https://www.figma.com/design/lFjxQoLHYzdLObC0g4HdWd?node-id=8994-44807&m=dev#1250829895) but it was resolved so I lost it